src/macro.ts

```diff
@@ -1,8 +1,11 @@
 import * as babel from '@babel/core'
+import annotateAsPure from '@babel/helper-annotate-as-pure'
 import { createMacro, MacroParams } from 'babel-plugin-macros'
 import { findJsxAttributeByName, getJsxAttributeName, getJsxAttributeValue } from './jsx-attribute'
 
-function twindMacro({ state }: MacroParams) {
+const localImportName = '_macro_tw'
```

Comment:
I think babel as a way to generate a local name that does not interfere with any existing name. It's a while ago ... I can not remember the function.

Reply:
Is it this? https://babeljs.io/docs/en/babel-helper-module-imports

Reply:
Looks like it. 👍

Reply:
Ah, I did a little digging (a.k.a stealing more ideas from twin.macro 😆) and it looks like this is a part of babel's core API: `program.scope.generateUid('tw')`